package.json

```diff
@@ -131,7 +131,8 @@
     "glob-parent": "^6.0.2",
     "parse-url": "^8.1.0",
     "graphql": "15.8.0",
-    "xml2js": "0.5.0"
+    "xml2js": "0.5.0",
+    "fast-xml-parser": "^4.4.1"
```

Comment:
aws-sdk has shipped a patch https://github.com/aws/aws-sdk-js-v3/pull/6330
the team noticed this applying the same resolution to amplify backend

Reply:
working on the fix now